lib/tasks/migrate_data.rake

```diff
@@ -0,0 +1,18 @@
+def create_commits!(repository)
+  repository.git.commits { |c| repository.commit_for!(c.oid) }
+end
+
+def attach_commit!(ontology_version)
+  commit = ontology_version.repository.commits.
+    find_by_commit_oid(ontology_version.commit_oid)
```

Comment:
Hmm, ok. Basically the reason for .@SOL is how the brain of ppl usually works, i.e. reading is usually: find the anchor (start) and than use pattern matching and stop if it appears, that we got it/can ignore the rest. So most of the time  "reading" doesn't actually switches on the region, which really _parses_ the text and stops much earlier wrt. end of line than we think ;-) - otherwise it would be too in-efficient, costs too much resources/power aka concentration and we would need a lot more good food ;-) . Anyway, with some training ... ;-)